### PR TITLE
Update readme with architecture doc list and links

### PR DESCRIPTION
**Pull request by sthurlock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,19 +13,19 @@ A work-in-progress application to simulate purchasing train tickets. This projec
...
+Our initial plan was to translate the requirements into a set of features, required and optional, as GitHub issues.  We started with a spreadsheet of the features and went through the provided design to determine the issues that existed with the code.  We decided the priority of the features & issues and our team members took up those issues.  Our architect created the documentation for the design including the component, sequence, and class diagrams for the system.
 
 We have been able to expand on the functionality of the system, but there are still many aspects that we did not have time to complete.  We discuss the issues we encountered in the next section.
 
-- [x] Scenario 3.1.1 - Search Tickets Scenario -- added Round Trip functionality
+- [X] Scenario 3.1.1 - Search Tickets Scenario -- added Round Trip functionality
```
Comment (GaelachAingeal) [inline README.md:20]: looks like this is a capital X but others are lower case. Not sure if you want to change it back to match the others.

File: README.md [hunk 2]
```diff
@@ -44,23 +37,57 @@ We have been able to expand on the functionality of the system, but there are st
...
+    * [Enter Traveler Info](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%205%20Enter%20Traveler%20Info%20Sequence%20Diagram%20SOA.pdf)
+    * [Enter Payment Info and Checkout](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%206%20Enter%20Payment%20Info%20and%20Checkout%20Sequence%20Diagram%20SOA.pdf)
+    * [Manage Order by Admin](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%207%20Manage%20Order%20By%20Admin%20Sequence%20Diagram%20SOA.pdf)
+    * [Manage Customer Info by Admin](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%208%20Manage%20Customer%20Info%20By%20Admin%20Sequence%20Diagram%20SOA.pdf)	
+    * [Request a Refund](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%209%20Request%20a%20Refund%20Sequence%20Diagram%20SOA.pdf)
+    * [View Order History](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%2010%20View%20Order%20History%20Sequence%20Diagram%20SOA.pdf)
```
Comment (GaelachAingeal) [inline README.md:54]: im not sure why but this diagram cuts off the right hand side. not sure if its just me or if its something that would need to be downloaded to view completely.

File: README.md [hunk 3]
```diff
@@ -44,23 +37,57 @@ We have been able to expand on the functionality of the system, but there are st
...
+* Quality attribute utility tree
+    * [Train Ticketing System QA Utility Tree](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Train%20Ticketing%20System%20QA%20Utility%20Tree.pdf)
+* Tactics applied for the quality attributes specified in the utility tree			
+    * [Modifiability Tactics](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Modifiability%20Tactics.pdf)
+    * [Security Tactics](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Security%20Tactics.pdf)
+    * [Performance Tactics](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Performance%20Tactics.pdf)
+    * [Usability Tactics](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Usability%20Tactics.pdf)		
+* Extras
+    * [Quality Attribute Tactics Scenarios](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Quality%20Attributes%20Tactics%20Scenarios.docx)
```
Comment (GaelachAingeal) [inline README.md:71]: not sure if we want to move this google docs or just leave as a download only.

**Review by GaelachAingeal — APPROVED** [review]
I only left minor comments but it can be merged as is

File: README.md (hunk 1 above)
Comment (sthurlock) [reply README.md:20]: fixed

File: README.md (hunk 2 above)
Comment (sthurlock) [reply README.md:54]: yes, we need a one page version of use case 10

File: README.md (hunk 3 above)
Comment (sthurlock) [reply README.md:71]: yes, need a pdf of this docx

**Pull request merged**